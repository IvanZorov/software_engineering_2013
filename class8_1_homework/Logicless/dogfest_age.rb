require 'csv'

godin = []
CSV.foreach("dogfest.csv") do |row|
	
	if (row[3].to_i >= ARGV[0].to_i) && (row[3].to_i <= ARGV[1].to_i)
		godin << row
	end
end

godin = godin.sort { |e2,e1| e2[3].to_f <=> e1[3].to_f }

CSV.open("dogfest_age_res.csv", "w") do |csv|
		godin.each do |e|
		csv << e
	
	end
end
