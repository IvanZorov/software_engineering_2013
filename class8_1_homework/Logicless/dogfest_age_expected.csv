31,CADENCE,Golden Retriever,5,Milko,Filipov
3,ATTILA,Havanese,5,Ivan,Zorov
37,CAJUN,Kuvasz,5,Phreak,Trinity
9,AURORA,Yorkshire Terrier,5,Ogi,Spasov 
80,JAVA,Pyrenean Shepherd,5,Kiril,Mitov
19,BABYDOLL,Siberian Husky,5,Georgi,Ivanov
20,BABYFACE,Siberian Husky,5,Ivan,Zorov,
29,CADBURY,German Shorthaired Pointer,5,Siv,Hd
75,JALAPENO,English Springer Spaniel,5,Debora,Iyall
11,AUSTIN,Soft Coated Wheaten Terrier,6,Preslav,Abarov
7,AUGUSTUS,Pointer,6,Viktor,Zorov
79,JASPER,Rat Terrier,6,Koi,Koiotov
66,FAUX,Finnish Lapphund,6,Amenei,Ata
22,BACCARAT,Petit Basset Griffon Vendeen,6,Sean,Paul
2,ATTICUS,Bloodhound,7,Athene,Live 
42,DANCER,American English Coonhound,7,Alex,Ich
77,JAMES,Plott,7,Makaio,Carino
98,PHOENIX,Rat Terrier,7,Thomas,Nastas
